src/main/java/seedu/sudohr/model/events/EventContainsEmployeePredicate.java

```diff
@@ -0,0 +1,30 @@
+package seedu.sudohr.model.events;
+
+import java.util.Set;
+import java.util.function.Predicate;
+
+import seedu.sudohr.model.person.Person;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
+ */
+public class EventContainsEmployeePredicate implements Predicate<Person> {
+    private final Set<Person> personsInEvent;
+
+    public EventContainsEmployeePredicate(Set<Person> personsInEvent) {
+        this.personsInEvent = personsInEvent;
+    }
+
+    @Override
+    public boolean test(Person person) {
```

Comment:
Is this your own manual test method?

@kennethk-1201 @jerrrren Y'all may want to avoid using Employee Equals method when testing if a person exists in an event / department. If im not wrong, in the scope of our app, we define two employees to be the same as long as their IDs are the same. So this check is handled by isSameEmployee.

The equals method of Employee right now checks every single field. Perhaps if i override the equals method, do y'all think its better? Still, technically there's nothing wrong with this strict equality definition uh, just not applicable to the definition of our app. Also `delete` commands use this Employee::equals as per the original AB3

Reply:
good point, we will include this after your merge